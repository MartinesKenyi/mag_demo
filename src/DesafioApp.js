import React from 'react';
import { AppRouter } from './routes/AppRouter';

export const DesafioApp = () => {
    return (
        <AppRouter />
    )
}
