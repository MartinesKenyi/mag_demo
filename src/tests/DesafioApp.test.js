import React from 'react';
import { shallow } from 'enzyme';
import { DesafioApp } from '../DesafioApp';

describe('Pruebas en <DesafioApp />', () => {
   
    test('Debe de mostrarse correctamente', () => {
        const wrapper = shallow( <DesafioApp /> );

        expect(wrapper).toMatchSnapshot();
    });
});
