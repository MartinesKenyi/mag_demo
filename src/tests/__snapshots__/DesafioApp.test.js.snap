// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <DesafioApp /> Debe de mostrarse correctamente 1`] = `
<UsuarioProvider>
  <SuscriptionProvider>
    <AppRouter />
  </SuscriptionProvider>
</UsuarioProvider>
`;
